pyproject.toml

```diff
@@ -2,5 +2,6 @@
 requires = [
     "setuptools>=45",
     "packaging>=20.0",
+    "typing_extensions",
```

Comment:
I had to add this to avoid a bug in this installation of setuptools_scm (Python 3.9).

Reply:
is this python 3.9 only or python 3.9+?

Reply:
I tried only with 3.9 and I had an import error without this line. However, I don't understand well how we are supposed to use [typing_extensions](https://pypi.org/project/typing-extensions/), since "The typing_extensions module serves two related purposes". I don't know if it is also useful for python > 3.9. It seems that it depends on what exactly is used in the code.

Reply:
@paugier ah, its my fault - i missed to make the _t import in hg/hg_git support optional, if you like you can replicate what i did for git,